debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordState.java

```diff
@@ -267,11 +267,11 @@ private Schema makeUpdatedSchema(List<FieldReference> additionalFields, Schema s
     }
 
     private SchemaBuilder updateSchema(FieldReference fieldReference, SchemaBuilder builder, Schema originalRecordSchema) {
-        return builder.field(fieldReference.getNewFieldName(), fieldReference.getSchema(originalRecordSchema));
+        return builder.field(fieldReference.getNewField(), fieldReference.getSchema(originalRecordSchema));
```

Comment:
Why is it here `getnewField` and not `getFieldName`? I think that method resolves the final (to be used) field name?

Reply:
Earlier itself it was with the `getNewField()` name and when I tried to change it to `getField()` I get many errors while running the JUnit `ExtractNewRecordStateTest`